spring-cloud-contract-tools/pom.xml

```diff
@@ -21,7 +21,8 @@
 		<module>spring-cloud-contract-converters</module>
 		<module>spring-cloud-contract-spec-pact</module>
 		<module>spring-cloud-contract-maven-plugin</module>
-		<module>spring-cloud-contract-gradle-plugin</module>
+		<!--<module>spring-cloud-contract-gradle-plugin</module>-->
```

Comment:
lol, why? Too slow? :P You can run with `-Pfast` to ignore the tests from that module